### PR TITLE
worked on login page

### PR DESCRIPTION
**Pull request by MitaliDeep17**

### CODE REVIEW

File: src/examples/LoginForm/LoginForm.tsx [hunk 1]
```diff
@@ -1,72 +1,114 @@
...
-          weight={"bold"}
-          align={"center"}
-          size={"3xl"}
-          className="mb-2"
-        >
-          Login
-        </Text>
+    <Box className=" w-[100vw] flex items-center justify-between">
+      <Box className="w-[50vw] flex justify-center flex-col items-center">
+      <Text as="label" weight={"black"}  className="absolute left-7 top-5 text-2xl">
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:12]: can you create enum for weight so we can use them everywhere

File: src/examples/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -1,72 +1,114 @@
...
-        </Text>
+    <Box className=" w-[100vw] flex items-center justify-between">
+      <Box className="w-[50vw] flex justify-center flex-col items-center">
+      <Text as="label" weight={"black"}  className="absolute left-7 top-5 text-2xl"> 
+            Logo
+          </Text>
+        <Box className="w-fit">
+          
+          <Stack className="relative">
+            <Text as="h2" weight={"bold"} size={"3xl"} className="mb-9" >
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:18]: use enum

File: src/examples/LoginForm/LoginForm.tsx [hunk 3]
```diff
@@ -1,72 +1,114 @@
...
+              size={"sm"}
+              weight={"medium"}
+              className="mb-1.5"
+            >
+              Username
+            </Text>
+            <Input
+              type="text"
+              id="username"
+              placeholder="Username"
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:34]: create a enum for placeholder.

File: src/examples/LoginForm/LoginForm.tsx [hunk 4]
```diff
@@ -1,72 +1,114 @@
...
+            </Text>
+            <Input
+              id="password"
+              type="password"
+              placeholder={"Password"}
+              style={webstyle.inputBoxDesign}
+            />
+            <Text
+              as="label"
+              htmlFor="password"
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:58]: htmlFor ???

File: src/examples/LoginForm/LoginForm.tsx [hunk 5]
```diff
@@ -1,72 +1,114 @@
...
+              size={"sm"}
+              weight={"bold"}
+              className="mb-1.5 w-full  text-end block text-[#7D5D4F]"
+
+            >
+              Forgot Password
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:64]: Don't use direct string

File: src/examples/LoginForm/LoginForm.tsx [hunk 6]
```diff
@@ -1,72 +1,114 @@
...
+              className="my-10"
+              style={webstyle.loginButton}
+            >
+              Login
+            </Button>
+
+            <Box className="flex w-[100%] justify-center ">
+              <IconButton
+                iconUrl={google}
+                backgroundColor="transparent"
```
Comment (deeporiontechnology) [inline src/examples/LoginForm/LoginForm.tsx:80]: if background color is transparent in every component 
then make it default

File: src/examples/LoginForm/LoginForm.tsx [hunk 7]
```diff
@@ -1,72 +1,148 @@
...
-          size={"sm"}
-          align={"center"}
-          className="mb-8"
-        >
-          Please enter your credentials to login
-        </Text>
+                <Text
+                  as="label"
+                  htmlFor="password"
+                  size={"sm"}
```
Comment (arshad0010) [inline src/examples/LoginForm/LoginForm.tsx:51]: use enum ?

File: src/examples/LoginForm/LoginForm.tsx [hunk 8]
```diff
@@ -1,72 +1,148 @@
...
+                  variant={ButtonVariant.DefaultPrimary}
+                  style={webstyle.loginButton}
+                >
+                  {Login.Login}
+                </Button>
+
+                <Text
+                  as="span"
+                  emphasis={"low"}
+                  size={"sm"}
```
Comment (arshad0010) [inline src/examples/LoginForm/LoginForm.tsx:86]: enum ?

File: src/examples/LoginForm/LoginForm.tsx [hunk 9]
```diff
@@ -1,72 +1,148 @@
...
+                    </Text>
+                  </Box>
+                </Box>
+              </Box>
 
+              <Box className="text-center">
+                <Text
+                  as="span"
+                  emphasis={"low"}
+                  size={"sm"}
```
Comment (arshad0010) [inline src/examples/LoginForm/LoginForm.tsx:120]: enum ?

File: src/components/PopupModal/index.tsx [hunk 10]
```diff
@@ -56,7 +56,7 @@ export const Modal = forwardRef<ModalProps, ModalPropss>(
 
     return (
       <div>
-        <PrimaryButton
```
Comment (naveenchoudhary129) [inline src/components/PopupModal/index.tsx:59]: don't use PrimaryButton,  use Button

File: src/components/PopupModal/index.tsx [hunk 11]
```diff
@@ -95,7 +95,7 @@ export const Modal = forwardRef<ModalProps, ModalPropss>(
               </p>
               <div className={`flex justify-end pt-6`}>
                 {button01 && (
-                  <PrimaryButton
```
Comment (naveenchoudhary129) [inline src/components/PopupModal/index.tsx:98]: use Button

File: src/components/PopupModal/index.tsx [hunk 12]
```diff
@@ -104,10 +104,10 @@ export const Modal = forwardRef<ModalProps, ModalPropss>(
                     onClick={toggleMenu}
                   >
                {buttontext1}
-                  </PrimaryButton>
+                  </Button>
                 )}
                 {button02 && (
-                  <PrimaryButton
```
Comment (naveenchoudhary129) [inline src/components/PopupModal/index.tsx:110]: use Button

File: src/components/PopupModal/index.tsx [hunk 13]
```diff
@@ -56,7 +56,7 @@ export const Modal = forwardRef<ModalProps, ModalPropss>(
 
     return (
       <div>
-        <PrimaryButton
+        <Button
           type="button"
```
Comment (naveenchoudhary129) [inline src/components/PopupModal/index.tsx:60]: Why do you give a button as a type

File: src/components/PopupModal/index.tsx [hunk 14]
```diff
@@ -104,10 +104,10 @@ export const Modal = forwardRef<ModalProps, ModalPropss>(
                     onClick={toggleMenu}
                   >
                {buttontext1}
-                  </PrimaryButton>
+                  </Button>
                 )}
                 {button02 && (
-                  <PrimaryButton
+                  <Button
                     type="button"
```
Comment (naveenchoudhary129) [inline src/components/PopupModal/index.tsx:111]: remove the type

File: src/examples/LoginForm/LoginForm.tsx [hunk 15]
```diff
@@ -1,72 +1,148 @@
...
+import { Login } from "./type";
+import { InputType } from "@/components/Input/type";
 
 export const LoginForm = () => {
   return (
-    <Box className="px-8 py-12 border border-gray-300 rounded-xl">
-      <Stack>
+    <Box className=" w-[100vw] flex items-center justify-between">
+      <Box className="w-[50vw] flex justify-center flex-col items-center">
         <Text
```
Comment (naveenchoudhary129) [inline src/examples/LoginForm/LoginForm.tsx:14]: use created Text from storybook

File: src/examples/LoginForm/LoginForm.tsx [hunk 16]
```diff
@@ -1,72 +1,148 @@
...
-          as="h2"
-          weight={"bold"}
-          align={"center"}
-          size={"3xl"}
-          className="mb-2"
+          as="label"
+          weight={"black"}
```
Comment (niteshMeena1) [inline src/examples/LoginForm/LoginForm.tsx:16]: can you make weight value in enum  .

File: src/examples/LoginForm/LoginForm.tsx [hunk 17]
```diff
@@ -1,72 +1,148 @@
...
+          className="absolute left-7 top-5 text-2xl"
         >
-          Login
+          Logo
```
Comment (niteshMeena1) [inline src/examples/LoginForm/LoginForm.tsx:19]: make children in enum .

File: src/examples/LoginForm/LoginForm.tsx [hunk 18]
```diff
@@ -1,72 +1,148 @@
...
+                  as="span"
+                  emphasis={"low"}
+                  size={"sm"}
+                  align={Alignment.Center}
+                >
+                  {Login.OtherPlatform}
+                </Text>
+                <Box className="flex flex-col gap-[0.94rem]">
+                  <Box className="flex items-center justify-center border rounded-lg border-[#72787F]">
+                    <IconButton iconUrl={google} className="w-fit"></IconButton>
```
Comment (niteshMeena1) [inline src/examples/LoginForm/LoginForm.tsx:93]: Icon button self close.

File: src/examples/LoginForm/LoginForm.tsx [hunk 19]
```diff
@@ -1,72 +1,148 @@
...
+                      className="w-fit text-[#72787F]"
+                      align={Alignment.Center}
+                      weight={TextWeight.Bold}
+                    >
+                      {Login.Google}
+                    </Text>
+                  </Box>
+
+                  <Box className="flex items-center justify-center border rounded-lg border-[#72787F]">
+                    <IconButton iconUrl={phone} className="w-fit"></IconButton>
```
Comment (niteshMeena1) [inline src/examples/LoginForm/LoginForm.tsx:104]: self Closed

**Pull request merged**